### DISCUSSION TITLE
Any plans to support windows arm64?

### DISCUSSION THREAD

**Discussion by afaq666** [0xd4d/dnlib#565] [Q&A]
Hi, I just wanted to ask if there is any plan to support native windows arm64 and if you have an estimate on when could that happen.

Thanks in advance.

**Comment by wtfsck** [comment]
What isn't supported? Some imports? I don't have windows arm64.

**Reply by afaq666** [reply]
thanks for your answer :)

I am talking about export methods: https://github.com/0xd4d/dnlib/blob/f618525d3f9e4733585610e1321f17b9fcb29307/src/DotNet/CpuArch.cs#L84
